docs/2.9.1/versions.json

```diff
@@ -0,0 +1,6 @@
+{
+  "daml": "2.9.1",
+  "canton": "2.9.1",
+  "daml_finance": "1.4.1",
```

Comment:
I don't understand why finance is 2.8.0 in assembly and docs is 1.4.1

Reply:
Yes, and 2.8.0 seemingly does not exist
```
Downloading daml-finance-doc-sources-2.8.0.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404
```

Reply:
Revert back to 1.4.1 and let the finance team follow up on it.